### ISSUE TITLE
Förderverein IP Block für gateway

### ISSUE THREAD

**Issue by noxilixon** [freifunk-berlin/meta#58]
Wir wollen einen Block vom Förderverein auf den neuen Verein übertragen und diesen dann zum Ausleiten ins Internet nutzen.

**Comment by nicolasberens** [comment]
Wir hatten ein gespräch und an sich sehen wir (Förderverein und ich) da kein problem.

Es gibt unbenutze IP ranges die extra hierfür gedacht waren.

Einzig wie das ganze Technisch geregelt wird müsste man klären:

- eigenes AS dass die range announced ?
- wir benutzen das förderverein AS und hinterlegen den verein als abuse, techc etc

**Comment by booo** [comment]
The new association should request an AS number from one of the Freifunk LIRs:

https://www.ripe.net/manage-ips-and-asns/as-numbers/request-an-as-number/

Does the new association already exist?

What is the size of the IP ranges we are talking about?

**Comment by nicolasberens** [comment]
@booo The problem/idea is, that the förderverein does not really want to run exit gateways.

Therefore the idea is to give the new verein a /24 to run this stuff and keep the blast radius limited

**Comment by booo** [comment]
> @booo The problem/idea is, that the förderverein does not really want to run exit gateways.

Yes, that is a fact for quite some time now. That they officially run quite a few exit gateways (BGP) is another confusing fact :)

Do they want to handover the whole IP range? Why only a /24? I guess we need more for our setup in multiple locations in Berlin.
